cryptorito/__init__.py

```diff
@@ -176,14 +176,26 @@ def gnupg_verbose():
     return ["-q"]
 
 
+def which_bin(cmd):
+    """Returns the path to a thing, or None"""
+    cmd = ["which", cmd]
+    try:
+        return stderr_output(cmd).strip().split('\n')[0]
+    except CryptoritoError:
+        return None
+
+
 def gnupg_bin():
-    """Return the path to the gpg binary"""
-    cmd = ["which", "gpg2"]
-    output = stderr_output(cmd).strip().split('\n')[0]
-    if not output:
-        raise CryptoritoError("gpg2 must be installed")
+    """Return the path to the gpg binary.
+    Note that on some systems this is gpg, and others gpg2. We
+    try to support both."""
+    gpg_output = which_bin("gpg2")
+    if not gpg_output:
```

Comment:
maybe consider a while loop over array of items to look for... i dislike nested ifs, but its a pure personal preference